apps/admin/pages/login.tsx

```diff
@@ -30,11 +30,11 @@ const Login: NextPage = () => {
     e.preventDefault();
     setSubmitting(true);
 
-    const res = await axios.get('http://localhost:8000/api/v1/auth', {
-   params: {
-      userName,
-     password,
-    },
+    const res = await axios.get(`${process.env.NEXT_PUBLIC_API_URL}/v1/auth`, {
+      params: {
+        userName,
+        password,
+      },
```

Comment:
please move this API call to another file in the API folder, like all the requests